.pre-commit-config.yaml

```diff
@@ -9,6 +9,7 @@ repos:
   rev: v4.0.1
   hooks:
   - id: trailing-whitespace
+    exclude: U_RHEL_7_V3R6_STIG_SCAP_1-2_Benchmark.xml
```

Comment:
Wondering whether we could exclude the whole `data` directory here

Reply:
yes, we can. excluded `tests/foreman/data/` directory.